apps/zbugs/api/index.ts

```diff
@@ -213,7 +212,7 @@ async function getQueriesHandler(
   await withAuth(request, reply, async authData => {
     reply.send(
       await handleGetQueriesRequest(
-        (name, args) => ({query: getQuery(authData, name, args)}),
+        (name, args) => getQuery(authData, name, args),
```

Comment:
This is a breaking change... Maybe I can undo this.

This was needed when Queries were `then`able.